pynipap/pynipap.py

```diff
@@ -100,7 +100,7 @@
 
         # Fetch VRF with ID 1 and print its name
         vrf = VRF.get(1)
-        print vrf.name
+        print(vrf.name)
```

Comment:
Py2.7 works with this type of print function instead of the print statement but by also importing from __future__ import print_function I think we disable the old type of print statements and guarantee that all prints use the print() function available in py3.